src/core_modules/capture-core/components/DataEntry/DataEntry.component.js

```diff
@@ -64,10 +62,6 @@ const styles = theme => ({
         paddingTop: theme.typography.pxToRem(10),
         paddingBottom: theme.typography.pxToRem(10),
     },
-    dataEntryFieldSection: {
-        padding: theme.typography.pxToRem(8),
-        maxWidth: theme.typography.pxToRem(892),
-    },
```

Comment:
Removing this for the same reason as the one above

![image](https://user-images.githubusercontent.com/4181674/100355090-1324db00-2fe9-11eb-9924-3f597b66ab89.png)